internal/util/util_test.go

```diff
@@ -1,280 +1,266 @@
 package util
 
```

Comment:
This is sort of hard to review. Can we separate the refactoring of all the unit tests to ginkgo/gomega in separate commits or a separate PR?